downstream/modules/platform/con_storage-options-for-operator-installation-on-ocp.adoc

```diff
@@ -0,0 +1,13 @@
+[id="con_storage-options-for-operator-installation-on-ocp_{context}"]
+= Storage options for {OperatorPlatform} installation on {OCP}
+
+If your installation scenario includes {HubName}, you must ensure that your storage option for {OperatorPlatform} has been set to `ReadWriteMany`.
```

Comment:
This is true if/when the customer is using "file" based storage. If they selected S3 or Azure than this access mode doesn't mean anything.

Reply:
Thanks for the feedback here. Was unsure about the technical accuracy. Will propose a new statement.